app/models/invoice.rb

```diff
@@ -31,6 +31,6 @@ def total_revenue
   end
 
   def self.incomplete
-    joins(:invoice_items).where.not(invoice_items: { status: 2 }).order(:created_at)
+    joins(:invoice_items).where.not(invoice_items: { status: 2 }).order(:created_at).uniq
```

Comment:
does this work? I think for AR it needs to be .distinct

Reply:
Agree with the .distinct.  Joel is fixing in his next PR